examples/timeseries_peak_finding.py

```diff
@@ -0,0 +1,78 @@
+"""
+==============================
+Find Peaks in SunPy TimeSeries
+==============================
+
+This example illustrates how to find minimum or maximum peaks in a TimeSeries.
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import sunpy.data.sample
+from sunpy.timeseries import TimeSeries
+
+##############################################################################
+# We will now create a TimeSeries object from an observational data source and 
+# do analysis on its truncated version.
```

Comment:
Reason for the truncation? Also give some dates or numbers to explain the truncation, is it 10% of the data, is it say year 1 to year 100?

Reply:
Its just for the appearance, plots on full timeseries looked cluttered. I will add the dates to explain it.